### PR TITLE
update for kairosdb 1.2.0

### PR DESCRIPTION
**Pull request by BrandonArp**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <aspectjrt.version>1.8.9</aspectjrt.version>
     <guice.version>3.0</guice.version>
     <guava.version>19.0</guava.version>
-    <kairosdb.version>1.1.3-1</kairosdb.version>
+    <kairosdb.version>1.2.0-1</kairosdb.version>
```
Comment (vjkoskela) [inline pom.xml]: Did you check that any shared transitives version match?

File: src/main/java/com/arpnetworking/kairosdb/aggregators/HistogramPercentileAggregator.java [hunk 2]
```diff
@@ -74,9 +72,22 @@ public String getAggregatedGroupType(final String groupType) {
...
     }
 
-
-    @javax.validation.constraints.Min(0)
-    @javax.validation.constraints.Max(1)
+    @NonZero
+    @FeatureProperty(
+            label = "Percentile",
+            description = "Data points returned will be in this percentile.",
+            default_value = "0.1",
```
Comment (vjkoskela) [inline src/main/java/com/arpnetworking/kairosdb/aggregators/HistogramPercentileAggregator.java:79]: This seems like an odd default value.

**Review by vjkoskela — COMMENTED** [review]
Just a couple of questions. Otherwise looks great!

File: pom.xml (hunk 1 above)
Comment (BrandonArp) [reply pom.xml]: Good catch. Updated Guice.

File: src/main/java/com/arpnetworking/kairosdb/aggregators/HistogramPercentileAggregator.java (hunk 2 above)
Comment (BrandonArp) [reply src/main/java/com/arpnetworking/kairosdb/aggregators/HistogramPercentileAggregator.java:79]: Yep, but that's the default for the non-histo one.

**Review by vjkoskela — APPROVED** [review]

**Pull request merged**